live-event-app/live-event-app/SceneDelegate.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  SceneDelegate.swift
+//
+//  PubNub Real-time Cloud-Hosted Push API and Push Notification Client Frameworks
+//  Copyright © 2021 PubNub Inc.
+//  https://www.pubnub.com/
+//  https://www.pubnub.com/terms
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+
+import PubNub
+import PubNubChat
+import PubNubChatComponents
+
+let PUBNUB_PUBLISH_KEY = "demo"
+let PUBNUB_SUBSCRIBE_KEY = "demo"
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+  
+  // Creates PubNub configuration
+  lazy var pubnubConfiguration = {
+    return PubNubConfiguration(
+      publishKey: PUBNUB_PUBLISH_KEY,
+      subscribeKey: PUBNUB_SUBSCRIBE_KEY,
+      userId: "user_0",
+      useSecureConnections: false,
+      origin: "127.0.0.1:8090"
```

Comment:
in case we will merge this branch to main, I think it should connect to production server instead of localhost